examples/cms-ghost/lib/api.js

```diff
@@ -0,0 +1,75 @@
+import GhostContentAPI from '@tryghost/content-api'
+
+const GHOST_API_URL = process.env.GHOST_API_URL || 'https://cms.gotsby.org'
```

Comment:
Are these defaults safe to include?

Reply:
In this case it is okay, because this is only an example site. It is the only option if we want users to be able to see something before providing custom keys. However, I obscured  the values with a base64 encoding. This is a very weak security measure but maybe better than plaintext.

Thanks a lot for reviewing!